docs/local-kiln.md

```diff
@@ -0,0 +1,43 @@
+---
+id: local-kiln
+title: Running Kiln Locally
+sidebar_label: Running Kiln Locally
+---
+
+To develop Kiln you need to run it locally. If you want to use the Vue Development Tools plugin in your browser, you will also need to run it locally because the Vue Development Tools only work when Vue is in development mode and Kiln is only in development mode when it is run locally.
+
+### Steps to run it locally
+
+1. Clone the [Clay Kiln GitHub Repository](https://github.com/clay/clay-kiln)
+2. cd into newly cloned clay-kiln directory and run the command `npm link`
+
+---
+The next 2 steps are only neccessary if you are actively developing Clay-Kiln, not if you just want a local version linked so you can use the Vue dev tools in your browser.
+
+  * While still in the clay-kiln directory, run `npm install`
+  * Start a watch on Kiln that will recompile as you make any changes to the code by running `npm run watch`
+---
+
+3. cd into the directory of the site running clay locally.
+    * Make sure you are in the directory that contains the node_modules directory for the site.
+    * Also make sure the site is running
+4. In this directory, run `npm link clay-kiln`
+5. Do a soft restart of your running site by typing `rs` inside the terminal where the site is running. ⚠️ Note that if you `ctrl-c` to stop your clay site completely and then start it again, you will no longer be linked to your local version of kiln. ⚠️ Whenever you do that you will need to run the `npm link clay-kiln` command again, and do a soft restart of your site instance.
+6. If you are ever unsure if you are locally linked to clay-kiln (or any other package) you can enter the command `ls -l node_modules | grep ^l` and it will list all locally linked packages. If you don't see clay-kiln listed, rerun the above 3 steps.
+7. A running site will sometimes cache its packages, so you may not see the local version of kiln being run in your browser right away.  You can try making a change to your sites code so that it recompiles, or try deleting the browserify-cache.json file and recompiling. That will usually do the trick.
```

Comment:
On `your sites code` can be `your site's code` or `the code of your site`. What do you think about this?